python/cdp/evm_server_account.py

```diff
@@ -193,6 +194,39 @@ async def sign_transaction(
             v=v,
         )
 
+    async def send_transaction(
+        self,
+        transaction_dict: TransactionDictType,
+        network: str,
+        idempotency_key: str | None = None,
+    ):
+        """Send a transaction.
```

Comment:
This method handles nonce management and gas estimation.

Reply:
Yeah, I understood it to do so, but was fighting the Web3.py library – it validates the transaction has all fields it deems necessary. May have to make our own, less strict, serializer

Reply:
Yes, it seemed that way. I'll give it another go

Reply:
Ok, I put some info in the docstring explaining that devs have to pass these params in. Will follow up with an improvement either later this week or next